authors/apps/profiles/views.py

```diff
@@ -175,3 +178,17 @@ def get(self, request):
             'followers': followers,
             'status': status.HTTP_200_OK}
         return Response(data=msg, status=status.HTTP_200_OK)
+
+
+class UserProfileListAPIView(generics.ListAPIView):
```

Comment:
Add a comment on this class to enlighten more on what it does.
Great work though.

Reply:
worked on.